examples/ucp_hello_world.c

```diff
@@ -92,18 +94,20 @@ static void set_msg_data_len(struct msg *msg, uint64_t data_len)
 
 static void request_init(void *request)
 {
-    struct ucx_context *ctx = (struct ucx_context *) request;
-    ctx->completed = 0;
+    struct ucx_context *contex = (struct ucx_context *)request;
+    contex->completed = 0;
```

Comment:
why needed?

Reply:
we use in other places `context` name, just aligned with other code